auto_cpufreq/core.py

```diff
@@ -42,9 +42,6 @@
 )
 CPUS = os.cpu_count()
 
-# ignore these devices under /sys/class/power_supply/
-POWER_SUPPLY_IGNORELIST = ["hidpp_battery"]
-
```

Comment:
I think its not needed anymore, but i am not sure

Reply:
I think we do still need it, check #332 PR for reference.